docs/2.0/reference/pipelines/feature-flags.md

```diff
@@ -11,10 +11,7 @@ Pipelines supports optional feature flags that allow you to enable or disable sp
 Enables all Pipelines experiment flags.
 </li>
 <li>
-**Default Value in Pipelines GitHub v3/GitLab v1**: Disabled
```

Comment:
Why get rid of the distinction from v3 to v4?

Reply:
This is PIPELINES_FEATURE_EXPERIMENT_ALL - its never enabled by default. This was incorrect information